Providers/nxOMSAutomationWorker/automationworker/worker/httpclientfactory.py

```diff
@@ -32,13 +34,16 @@ def __init__(self, cert, key, insecure=False):
 
         # set proxy if valid proxy_configuration path is detected
         proxy_conf_path = configuration.get_proxy_configuration_path()
-        if configuration.get_proxy_configuration_path() != configuration.DEFAULT_PROXY_CONFIGURATION_PATH \
+        if proxy_conf_path != configuration.DEFAULT_PROXY_CONFIGURATION_PATH \
```

Comment:
Can you send me a link to where `get_proxy_configuration_path()` and `DEFAULT_PROXY_CONFIGURATION_PATH` are defined? Is this the OMSAgent proxy, or machine proxy?